model/src/main/java/com/javahelp/model/user/ProviderUserInfo.java

```diff
@@ -8,22 +8,45 @@ public class ProviderUserInfo extends UserInfo {
     private boolean certified;
     private String address;
     private String phoneNumber;
+    private Gender gender;
```

Comment:
Just a thought, since we have the attribute as `practiceName` for providers, do we allow a company/organization of therapists to register one account? If yes then it does not make sense to have a gender for this company. But if no, I think we should change `practiceName` into `firstName` and `lastName` just like the client.

Reply:
Wait, I thought I designed to assign `Gender` as an attribute to a Provider User, not a company?